src/HttpClient.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
is this compatible with PHP 5?

Reply:
It seems OK to keep it, no functional errors or PHPStan errors. But we can remove those declarations if you prefer.